### ISSUE TITLE
[Feature] Add more quizzes

### ISSUE THREAD

**Issue by RehanChalana** [JAYESHBATRA/Virtuo-Learn#27]
### Description

different quizzes implementation is almost same , i just need to vary questions  in script.js file and create a folder for the new quiz using the existing quiz , i have sufficient knowledge of science so maybe i can help in adding one or more quiz , let me know if i can work on it

### Checklist

- [X] I have starred the repository
- [X] I have checked the existing issues
- [X] I have read the discussion tab thoroughly and got the project idea 
- [X] I am willing to work on this issue (optional)
- [X] I am a SWOC'23 contributor

**Comment by RehanChalana** [comment]
hey i wanted to ask i was working on it , when i have to show the recommended videos for the weak topics... 
I don't have thumbnails with the topic names 
I am adding a quiz for laws of motion and its topics 
![image](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/139042983/162e2b8e-ac26-42e7-9f50-d647a6bf330e)




also wanted to ask should one quiz each pr 
or can i directly add 2 or more quizzes in a single pr

**Comment by RehanChalana** [comment]
sorry i accidently closed issue 
I needed some help on the part where i have to show the recommended vidoes
I don't have proper thumbnails for the topics i am working on
![image](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/139042983/d43b7203-d88d-42df-9a8a-e48c5ff0dfad)
Here i need thumbnails with the quiz topic i am working on

**Comment by JAYESHBATRA** [comment]
First make a pull request in which you have added one quiz so that if any changes are required it can be done. Once the pull request is merged then you can raise another issue for the remaining quizes. For thumbnails, you have to make demo thumbnails which can be linked with the tag of the questions.

**Comment by JAYESHBATRA** [comment]
@RehanChalana you can use https://coverview.vercel.app/ for making demo thumbnails. 
<img width="960" alt="image" src="https://github.com/JAYESHBATRA/Virtuo-Learn/assets/112322302/10863f81-6efc-4ef4-8ba4-cbbe41e5da2f">

**Comment by RehanChalana** [comment]
![cover](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/139042983/fe5c8c0e-7a5f-430b-ad46-cc7c9f8d78d8)
hey i understood 
i made this dummy thumbnail if you could check

**Comment by JAYESHBATRA** [comment]
Great ! No issues with this thumbnail. You can go ahead.
